shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.h

```diff
@@ -10,6 +10,9 @@
 #include "flutter/shell/platform/darwin/common/framework/Headers/FlutterChannels.h"
 #include "flutter/shell/platform/darwin/ios/framework/Source/FlutterTextInputDelegate.h"
 
+#if FLUTTER_RUNTIME_MODE == FLUTTER_RUNTIME_MODE_DEBUG
```

Comment:
FYI: This will be unnecessary after https://github.com/flutter/engine/pull/17624 lands

Reply:
I was talking about the "FLUTTER_EXPORT" not being needed once that lands.  We used to have to export things we wanted to tests, now tests live with the code so it isn't necessary.